docs/source/welcome/services/index.md

```diff
@@ -0,0 +1,64 @@
+(servicesprovided)=
+# Services we Provide
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`device-desktop;1.5em;screen-full` Compute
+:link: ../../hardware/hardware_overview
+:link-type: doc
+
+An overview of our computational resources.
```

Comment:
```suggestion
An overview of available computational resources.
```
If "available" doesn't make sense, you can just remove that, or change to something else. (Per ITS Comms, we aren't supposed to use we/our)